appveyor.yml

```diff
@@ -9,6 +9,8 @@ matrix:
   allow_failures:
     - configuration: MINGW_64-gcov
 install: []
+before_build:
+- ps: Install-Product node 8
```

Comment:
This takes 20s according to appveyor logs. But it doesn't mention a path, so we can't add it to cache...